src/api/assets.js

```diff
@@ -1,174 +1,40 @@
-import { Operation, xdr } from 'tokend-js-sdk'
-
-import config from '../config'
-import server from '../utils/server'
-import store from '../store'
-import { ServerCallBuilder } from './ServerCallBuilder'
-import { envelopOperations } from './helpers/envelopOperations'
-
-const ScopedServerCallBuilder = ServerCallBuilder.makeScope()
-.registerResource('assets')
-.registerResource('holders')
-.registerResource('asset_pairs', 'assetPairs')
-.registerResource('transactions')
-.registerResource('convert')
+import { Sdk } from '@/sdk'
 
 export default {
-  getAll () {
-    return new ScopedServerCallBuilder()
-      .assets()
-      .sign()
-      .get()
-  },
-
-  getAllSystemAssets () {
-    return new ScopedServerCallBuilder()
-      .assets()
-      .sign()
-      .get({ owner: config.MASTER_ACCOUNT })
-  },
-
-  get (code) {
-    return new ScopedServerCallBuilder()
-      .assets(code)
-      .sign()
-      .get()
-  },
-
-  getPairs () {
-    return new ScopedServerCallBuilder()
-      .assetPairs()
-      .get()
-  },
-
-  getHolders (code) {
-    return new ScopedServerCallBuilder()
-      .assets(code)
-      .holders()
-      .sign()
-      .get()
-  },
-
   createPair (params) {
-    const tx = envelopOperations(
-      Operation.manageAssetPair({
-        base: params.base,
-        quote: params.quote,
-        action: xdr.ManageAssetPairAction.create(),
-        policies: +params.policies,
-        physicalPrice: '' + params.physicalPrice,
-        physicalPriceCorrection: '' + params.physicalPriceCorrection,
-        maxPriceStep: '' + params.maxPriceStep,
-        source: config.MASTER_ACCOUNT
-      })
-    )
-    return new ScopedServerCallBuilder()
-      .transactions()
-      .sign()
-      .post({ tx })
+    const operation = Sdk.base.Operation.manageAssetPair({
+      base: params.base,
+      quote: params.quote,
+      action: Sdk.xdr.ManageAssetPairAction.create(),
+      policies: +params.policies,
+      physicalPrice: '' + params.physicalPrice,
+      physicalPriceCorrection: '' + params.physicalPriceCorrection,
+      maxPriceStep: '' + params.maxPriceStep
+    })
+    return Sdk.horizon.transactions.submitOperations(operation)
```

Comment:
inconsistent stuff. 

look, `manageMaster` does: 

```
const response = await Sdk.horizon.transactions.submitOperations(operation)
return response.data
```

when this method returns the whole response. 

Let's choose only 1 way to return the response